pkg/sources/fake/fake.go

```diff
@@ -6,15 +6,36 @@ import (
 
 	"telemetry/pkg/msgs"
 	"telemetry/pkg/pubsub"
+	"telemetry/pkg/util/randutil"
 )
 
 const (
 	// interval between fake data readings
-	readInterval = 20 * time.Millisecond
+	readInterval = 200 * time.Millisecond
+
+	// TODO(karlding): change these to the actual values
+	rightTurnOn     = 1
```

Comment:
Yeah, I'll switch this over to use the protobuf messages after this. In the meantime, I think it's fine to use this (since the messages are still up in the air), and then use the example asciiprotobuf file to compile some protos so we basically have all the code ready for the actual asciiprotobufs